storage/watchable_store_test.go

```diff
@@ -222,3 +234,95 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestMapWatchingToEvents(t *testing.T) {
```

Comment:
@xiang90 
I changed the code to not create watchableStore. But test table method seems too manual in this case.
Please let me know if you have any feedback. Thanks,